src/main/java/com/github/amitsureshchandra/onlinecompiler/service/lang/CommonLangService.java

```diff
@@ -39,32 +60,62 @@ public OutputResp run(CodeReqDto dto) throws IOException, InterruptedException {
 
         // creating a container
         String containerName = UUID.randomUUID().toString();
-        String command = dockerService.getDockerCommand(userFolder, dto.getCompiler(), containerName);
-        log.info("command : " + command);
 
-        // running shell service &
-        OutputResp outputResp = shellService.run(command);
+        CreateContainerResponse container =  dockerClient
+            .createContainerCmd(dockerService.containerMap.get(dto.getCompiler()))
+                .withHostName(containerName)
+                .withHostConfig(new HostConfig().withCpuCount(1L).withCpuPercent(100L).withMemory(104857600L).withBinds(new Bind( hostTempFolder + userFolder, Volume.parse("/opt/myapp"), AccessMode.rw)))
```

Comment:
need to add feature to maintain this config because it is different for each docker image, or we need to make same volume path

Reply:
for now using this as common config for all containers